google/cloud/aiplatform/metadata/utils.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import List, Optional, Union
+
+from google.cloud.aiplatform import base
+
+
+def make_gcp_resource_url(resource: base.VertexAiResourceNoun) -> str:
```

Comment:
Nit: This seems like it may be useful for resources other than metadata. Maybe move it to the module's main utils?

Reply:
done